portfolio/src/main/webapp/script.js

```diff
@@ -55,18 +55,34 @@ function createListElement(text) {
 }
 
 /** Fetches login status. */
-function displayCommentForm() {
+function displayLoginContent() {
   fetch('/login-status').then(response => response.json()).then((info) => {
 
     const comment_body = document.getElementById('comment-form');
     const login_button = document.getElementById('login-button-1');
-    if (info[0] === "true") {
-        comment_body.style.display = "block";
-        login_button.style.display = "none";
+
+    const item_element = document.getElementById('login-header');
+    const a = document.createElement('a');
+
+    if (info["status"] === "true") { // User is logged in
```

Comment:
One tip for JavaScript: info["status"] is the same as info.status so you can do that instead.